src/pages/FavoritesPage/FavoritesPage.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { FavContext } from '../../components/FavContext';
+import { Loader } from '../../components/Loader/Loader';
+import { ProductPage } from '../../components/ProductPage';
+import { getProducts } from '../../helpers/getProducts';
+import { Product } from '../../types/Product';
+
+export const FavoritesPage: React.FC = () => {
+  const { favs } = useContext(FavContext);
+  const [products, setProducts] = useState<Product[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    (async () => {
```

Comment:
let's use then everywhere instead of this wrapper!)

Reply:
done, TY )